pkg/reconciler/v1alpha1/configuration/resources/revision.go

```diff
@@ -69,17 +69,17 @@ func UpdateRevisionLabels(rev *v1alpha1.Revision, config *v1alpha1.Configuration
 	}
 }
 
-// RevisionLabelValueForKey returns the label value for the given key
+// RevisionLabelValueForKey returns the label value for the given key.
 func RevisionLabelValueForKey(key string, config *v1alpha1.Configuration) string {
 	switch key {
 	case serving.ConfigurationLabelKey:
 		return config.Name
 	case serving.ServiceLabelKey:
 		return config.Labels[serving.ServiceLabelKey]
 	case serving.DeprecatedConfigurationGenerationLabelKey:
-		return fmt.Sprintf("%v", config.Spec.Generation)
+		return fmt.Sprintf("%d", config.Spec.Generation)
```

Comment:
FYI - you don't need to be explicit 

From: https://golang.org/pkg/fmt/
> %v	the value in a default format

Reply:
I know that %v is catchall, but, fwiw, I believe being explicit is more useful (e.g. I had to go and look up the definition of the Generation to find its defined type).